src/containers/addNew.js

```diff
@@ -0,0 +1,193 @@
+import React, { Component } from 'react';
+import Ratings from 'react-ratings-declarative';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import './form.scss';
+import { addNewBeer } from '../actions';
+
+class AddNew extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      touched: {
+        beerName: false,
+        rating: false,
+        desc: false,
+      },
+      beerName: '',
+      rating: 1,
+      desc: '',
+      file: '',
+      isDisabled: true,
+    };
+  }
+
+  changeTheRating = newRating => {
+    this.setState({ rating: newRating });
+  };
+
+  handleChange = field => event => {
+    this.setState(
+      {
+        [field]: event.target.value,
+      },
+      () => {
+        if (this.state.beerName !== '' && this.state.desc !== '') {
+          this.setState({
+            isDisabled: false,
+          });
+        } else
+          this.setState({
+            isDisabled: true,
+          });
+      },
+    );
+  };
+
+  validate = field => () => {
+    this.setState({
+      touched: { ...this.state.touched, [field]: true },
+    });
+  };
+
+  renderField = (label, field) => {
+    let classnames = null;
+
+    if (field === 'beerName')
+      classnames = classNames('form-group', {
+        'has-danger': this.state.touched.beerName && this.state.beerName === '',
+      });
+    else
+      classnames = classNames('form-group', {
+        'has-danger': this.state.touched.desc && this.state.desc === '',
+      });
+
+    return (
+      <div className={classnames}>
+        <label className="field-label">{label}</label>
+        <input
+          className="form-control"
+          type="text"
+          value={field === 'beerName' ? this.state.beerName : this.state.desc}
+          onChange={this.handleChange(field)}
+          onBlur={this.validate(field)}
+        />
+      </div>
+    );
+  };
+
+  renderNumberField = field => {
+    return (
+      <div className="form-group" id="rating-field">
+        <label className="field-label">Enter the rating</label>
+        <label className="stars">
+          <Ratings
+            rating={this.state.rating}
+            widgetEmptyColors="rgb(255,239,212)"
+            widgetHoverColors="rgb(173, 21, 21)"
```

Comment:
move color variables to sobie variables/theme/config/top of the file, dont store them inline